docs/kyma/04-07-local-installation-deprecated.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Install Kyma locally (Scripts)
+type: Installation
+---
+
+>**WARNING:** Will be deprecated {TBD} with Kyma release 1.14
```

Comment:
We can't really have a {TBD} and a message that basically says that "it's gonna stop working some day". If the CLI is functional right now, then switch to it right now. Keeping two installation instructions living side by side makes this simple process more complicated than it should be. 

For me, there should be a single document describing the installation. Otherwise all of the simplification efforts taken recently basically go to waste and we're back in ConfuseTown.

Reply:
The idea is to have it deprecated since 1.4 and then delete it. So giving the people a smoother adoption phase instead of having a hard cut. It will be deleted with 1.4. That should be the message, nothing TBD.